### DISCUSSION TITLE
Option for importing embeddings

### DISCUSSION THREAD

**Discussion by xubairp** [n4ze3m/dialoqbase#208] [General]
Hello,

I have 2 questions:

1. How to access the Postgres/PGvector instance installed with the diagloqbase, is there a specific url/interface ? basically to verify/access the data/queries.
2. I am using supabase pgvector db to store my embeddings, is there a way to import my existing embeddings from supabase/pgvector into the dialogbase ? it would be amazing to "import embeddings" as an option under the "ADD NEW DATA SOURCE".

**Comment by n4ze3m** [comment]
> How to access the Postgres/PGvector instance installed with the diagloqbase, is there a specific url/interface ? basically to verify/access the data/queries.

Currently, no; however, it can be added in the upcoming release as a new feature.

> I am using supabase pgvector db to store my embeddings, is there a way to import my existing embeddings from supabase/pgvector into the dialogbase ? it would be amazing to "import embeddings" as an option under the "ADD NEW DATA SOURCE".

Importing and exporting embeddings is currently not possible for Dialoqbase. I will look into it

**Reply by xubairp** [reply]
cool, i believe this feature will make a huge impact, specially for those who spent significant time on generating the embedding for large dataset. with this feature it will be easy to import their existing embedding and run bot.

looking forward to next release
